### PR TITLE
skbio version 0.5.5 changelog

### PR DESCRIPTION
**Pull request by mortonjt**
Please complete the following checklist:

* [x] I have read the guidelines in [CONTRIBUTING.md](https://github.com/biocore/scikit-bio/blob/master/CONTRIBUTING.md).

* [x] I have documented all public-facing changes in [CHANGELOG.md](https://github.com/biocore/scikit-bio/blob/master/CHANGELOG.md).

* [ ] **This pull request includes code, documentation, or other content derived from external source(s).** If this is the case, ensure the external source's license is compatible with scikit-bio's [license](https://github.com/biocore/scikit-bio/blob/master/COPYING.txt). Include the license in the `licenses` directory and add a comment in the code giving proper attribution. Ensure any other requirements set forth by the license and/or author are satisfied. **It is your responsibility to disclose code, documentation, or other content derived from external source(s).** If you have questions about whether something can be included in the project or how to give proper attribution, include those questions in your pull request and a reviewer will assist you.

* [x] **This pull request does not include code, documentation, or other content derived from external source(s).**

**Note:** [REVIEWING.md](https://github.com/biocore/scikit-bio/blob/master/REVIEWING.md) may also be helpful to see some of the things code reviewers will be verifying when reviewing your pull request.

Note that this should be the final changelog before the scikit-bio 0.5.5 release.  @thermokarst @ebolyen @wasade @ElDeveloper

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +22,15 @@
 
 ### Miscellaneous
 
+* A pytest runner is shipped with every installation ([#1633](https://github.com/biocore/scikit-bio/pull/1633))
+
+* The nosetest framework has has been replace in favor of pytest ([#1624](https://github.com/biocore/scikit-bio/pull/1624))
```
Comment (ElDeveloper) [inline CHANGELOG.md]: has has -> has
replace -> replaced

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +22,15 @@
...
+
+* The numpy docs are deprecated in favor of napoleon ([#1629](https://github.com/biocore/scikit-bio/pull/1629))
```
Comment (ElDeveloper) [inline CHANGELOG.md]: I think I would change this to be `* The documentation is now built using [Napoleon](http://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html) .......`.

File: CHANGELOG.md [hunk 3]
```diff
@@ -20,6 +22,15 @@
...
+
+* This version is now compatiable with pandas 0.23 ([#1627](https://github.com/biocore/scikit-bio/pull/1627))
```
Comment (ElDeveloper) [inline CHANGELOG.md]: Can you also mention NumPy? Maybe mention the ranges themselves, something like:

` This version is now compatible with NumPy >= 1.9.2 and Pandas >= 0.23`.

**Comment by mortonjt** [comment]
Thanks for the feedback @ElDeveloper !  

I now passed into the changelog that there is compatibility with py3.6 and py3.7
However, looking at the travis log, it looks like these versions of python are throwing errors in the docstrings: https://travis-ci.org/biocore/scikit-bio/jobs/465455313

I'm guessing that we should fix these issues in this release.

**Comment by ElDeveloper** [comment]
👍 Looks good to me!

**Comment by ElDeveloper** [comment]
I missed your message. Yeah, it looks like there's a few flake8 errors 😞

**Comment by mortonjt** [comment]
Yes, but the flake8 errors seem to be specific to python 3.6 and 3.7. Not sure why -- I'm double checking the napolean docs to make sure those are rendered properly

**Comment by mortonjt** [comment]
Ok, I'm disabling W605 -- really not sure how you can get working regex tests with this weird limitation.

Otherwise, this should be good to go. @ElDeveloper want to merge this in after the tests pass?

**Comment by ElDeveloper** [comment]
I don't think it is a good idea to disable W605, since this is a rule that affects newer versions of Python. I also wouldn't be in favor of dropping support for python 3.5 based on a flake8 failure. This is the error in the 3.5 build:

```
flake8 skbio setup.py checklist.py --extend-ignore=W605
Usage: flake8 [options] file file ...
flake8: error: no such option: --extend-ignore
```

**Comment by mortonjt** [comment]
Ok that's fine. Could you look at one of the regular expressions then? It
appeared to be a bit of a catch 22

**Comment by ElDeveloper** [comment]
Yes, I'll have a look in a bit. :+1:

**Comment by mortonjt** [comment]
@antgonza or @wasade could either of you double check this?

File: skbio/io/format/tests/test_stockholm.py [hunk 4]
```diff
@@ -853,8 +853,8 @@ def test_gr_wrong_dataframe_item_length_error(self):
                               positional_metadata=pos_metadata_dataframe)])
         with self.assertRaisesRegex(StockholmFormatError,
                                     r'Sequence-specific positional metadata.*'
-                                    'must contain a single character.*Found '
-                                    'value\(s\) in column AC'):
+                                    r'must contain a single character.*Found '
```
Comment (antgonza) [inline skbio/io/format/tests/test_stockholm.py]: It seems like the error messages are missing a space. For example:
- here: `character.*Found`
- the test below `character.*Found value`
is that right? If it is, can we fix?

Comment (mortonjt) [reply skbio/io/format/tests/test_stockholm.py]: @antgonza note that this is a regex check -- that * is ignoring characters between the `.` and `Found`

Comment (antgonza) [reply skbio/io/format/tests/test_stockholm.py]: duh! thanks!

**Pull request merged**